### PR TITLE
Subtype

### PR DESCRIPTION
**Pull request by t9haja00**

### CODE REVIEW

File: application/views/subtypes/main.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        echo '<br> Price in credits'.$row['cost'];
+        echo '<br> info'.$row['address'];
+        echo '<br>';
+        echo '</div>';
+        // here we can put to link to buy page
+        echo '<br>';
+
+        // would echo the ID so each button has the Id wont work? 
+
+        echo '<br> <button id="echo$row[\'subtype_id\'];"type="button" onclick="test(this.id)" class="btn btn-info">Buy</button>';
```
Comment (t9haja00) [inline application/views/subtypes/main.php:33]: How to fix So id= will be the subtype id. 
If have some other smart way to do it won't mind.

File: application/models/Sub_type.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+    class Sub_type extends CI_model
+    {
+        public function __construct()
+        {
+            //Codeigniter : Write Less Do More
+            parent::__construct();
+
+            // Load CodeIgniter Libary for databases
+            $this->load->database();
```
Comment (k-rahman) [inline application/models/Sub_type.php:11]: Database library already autoloading.

File: application/views/subtypes/main.php [hunk 3]
```diff
@@ -0,0 +1,56 @@
+<?php session_start(); ?>
```
Comment (k-rahman) [inline application/views/subtypes/main.php:1]: Session library is already autoloading.

File: application/views/subtypes/main.php [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    </div>
+
+
+
+</div>
+
+<!-- For bootstrap links -->
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+<!-- Script what could be used to get the button Id put in session variable or flastdata -->
```
Comment (k-rahman) [inline application/views/subtypes/main.php:48]: Already exists in footer.php.

File: application/views/subtypes/main.php (hunk 1 above)
Comment (k-rahman) [reply application/views/subtypes/main.php:32]: After getting your data from the table store it the id on a session. I hope that what you wanted. I'm not sure though what do you mean by id = will be the subtype id.

**Pull request merged**